workflows/logon-details/LogonDetails.py

```diff
@@ -586,7 +653,8 @@ def main():
             cid=options["cid"],
             rating=options["rating"],
             password=options["password"],
-            cpdlc=options["cpdlc"]
+            cpdlc=options["cpdlc"],
+        vccs_ptt=options.get("vccs_ptt_scan_code", "")
```

Comment:
Can you indent this the same as the others for style/consistency?
```suggestion
            vccs_ptt=options.get("vccs_ptt_scan_code", "")
```